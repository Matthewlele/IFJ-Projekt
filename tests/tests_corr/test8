def vratJednicku ()
  1
end

if 1 == 1 then do
  print vratJednicku
end
