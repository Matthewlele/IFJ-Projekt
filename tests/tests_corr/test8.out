SYNTAX ERR: row 5
